src/index.js

```diff
@@ -175,10 +163,15 @@ export default function inject(options) {
         }
       });
 
-      const keys = Object.keys(newImports);
-      if (!keys.length) return null;
+      if (newImports.size === 0) {
+        return {
+          code,
+          ast
+        };
+      }
+      const importBlock = Array.from(newImports.values())
+        .join("\n\n");
 
-      const importBlock = keys.map(hash => newImports[hash]).join("\n\n");
       magicString.prepend(importBlock + "\n\n");
 
       return {
```

Comment:
Question, i am not retuning the AST here on purpose, since the original code was transformed, is it safe to pass it?

Reply:
If the original code was not modified, definitely return the AST! Otherwise of course this can result in Rollup messing everything up as it will always assume the returned AST corresponds to the returned code.